### ISSUE TITLE
How to use darknet as a plugin？

### ISSUE THREAD

**Issue by ApacheWang** [Xingtao/FFdynamic#3]
Hi, Xingtao, thank you for your answer. I am sorry to bother you. I understand that your cvDnnDetect.cpp is the image recognition part, but I don't understand how to do it. Can you recommend some information?

**Comment by ApacheWang** [comment]
I think if this works, it would be a very good project. That's so cool!

**Comment by Xingtao** [comment]
I will try work out a skeleton of darknet plugin this weekend.  It is not hard, basically the same as example here:
https://github.com/Xingtao/FFdynamic/blob/master/examplePlugin/README.md

**Comment by ApacheWang** [comment]
Hi, Xingtao, I spent 2 hours writing a python program that easily implemented the functionality I wanted. I found that using Python makes things easier. My idea is this: first define an rtmp stream, let opencv handle, and cut the stream into a frame, then define a ffmpeg instruction, and listen through the pipeline form, call yolov3 model through python to identify the frame, and I launched it into the live stream of nginx, so I can easily see the results I want. I mainly use tensorflow to accelerate the recognition process. I found that the real-time effect can be approximated on the M40.
